corehq/apps/users/templates/users/web_users.b3.html

```diff
@@ -250,7 +340,7 @@ <h3 class="panel-title">
                         <h4 class="modal-title">{% trans "Delete invitation" %}</h4>
                     </div>
                     <div class="modal-body">
-                        <h4> {% trans "Are you sure you want to delete this invitation?" %}</h4>
+                        <p>{% trans "Are you sure you want to delete this invitation?" %}</p>
                     </div>
                     <div class="modal-footer">
                         <button type="button" class="btn btn-default" data-dismiss="modal">{% trans "Cancel" %}</button>
```

Comment:
I think this is my first time reading angular stuff - it's remarkably straightforward.